mtconnect-examples/src/main/java/io/github/protocol/mtconnect/examples/MTConnectIoTDAServerExample.java

```diff
@@ -1,13 +1,33 @@
 package io.github.protocol.mtconnect.examples;
 
+import io.github.openfacade.http.HttpClientConfig;
 import io.github.openfacade.http.HttpServerConfig;
 import io.github.openfacade.http.HttpServerEngine;
+import io.github.protocol.mtconnect.api.MTConnectDevices;
+import io.github.protocol.mtconnect.client.MTConnectClient;
+import io.github.protocol.mtconnect.client.MTConnectClientConfiguration;
+import io.github.protocol.mtconnect.common.XmlUtil;
 import io.github.protocol.mtconnect.server.MTConnectServer;
 import io.github.protocol.mtconnect.server.MTConnectServerConfiguration;
 import io.github.protocol.mtconnect.server.impl.IoTDAMtProcessor;
 
+
 public class MTConnectIoTDAServerExample {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws Exception {
+        startServer();
+
+        MTConnectClientConfiguration configuration = new MTConnectClientConfiguration();
+        configuration.setHost("127.0.0.1");
+        configuration.setPort(36633);
+        HttpClientConfig httpClientConfig = new HttpClientConfig.Builder().build();
+        configuration.setHttpConfig(httpClientConfig);
+        MTConnectClient mtConnectClient = new MTConnectClient(configuration);
+
+        MTConnectDevices devices = mtConnectClient.devices();
+        System.out.println(XmlUtil.toXml(devices));
```

Comment:
use `log.info` instead

Reply:
At first, I thought that `example` should use `printIn` to print to the console for more intuitive use, while log4j is more for running logs and other outputs. Now I am a little hesitant. We can discuss this issue in detail.